### ISSUE TITLE
Use collision model for tar_cover props

### ISSUE THREAD

**Issue by zlsa** [Terri00/CS-GO-Auto-Radar#19]
It would be great to have an option to use the collision model for props in `tar_cover`. Some props are very complex, when all you need is a vague outline.

**Comment by Terri00** [comment]
Yeah, that would make sense especially for props such as trees. Where you only need the trunk to be marked.

I'll take a look and see if the .phy format is easy to incorporate. Will leave this open until I find some info :)

Thanks!
